### PR TITLE
Fix travis script 

### PR DESCRIPTION
**Pull request by PatriceJiang**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -175,3 +175,6 @@ metal-support*.zip
 **/org.eclipse.buildship.core.prefs
 
 desktop.ini
+
+
+tools/travis-scripts/test.sh
```
Comment (minggo) [inline .gitignore]: Is it a mistake modification?

Comment (PatriceJiang) [reply .gitignore]: yes, I will remove this

File: tools/travis-scripts/run-script.sh [hunk 2]
```diff
@@ -151,6 +151,7 @@ function generate_pull_request_for_binding_codes_and_cocosfiles()
     local COCOS_BRANCH="update_lua_bindings_$ELAPSEDSECS"
     local COMMITTAG="[ci skip][AUTO]: updating luabinding & cocos_file.json automatically"
     local PULL_REQUEST_REPO="https://api.github.com/repos/cocos2d/cocos2d-x/pulls"
+    local COCOSFILE_PATH="$COCOS2DX_ROOT/templates/cocos2dx_files.json"
```
Comment (minggo) [inline tools/travis-scripts/run-script.sh]: I don't quite understand what's the error. Is it because the label is used without definition?

Comment (minggo) [reply tools/travis-scripts/run-script.sh]: COCOSFILE_PATH is used in `update_cocos_files()` too, so i think should declare this variable at the begin as COCOS2DX_ROOT.

Comment (PatriceJiang) [reply tools/travis-scripts/run-script.sh]: yes, it is not declared in the function's scope

File: .gitignore [hunk 3]
```diff
@@ -175,3 +175,4 @@ metal-support*.zip
 **/org.eclipse.buildship.core.prefs
 
 desktop.ini
+
```
Comment (PatriceJiang) [inline .gitignore]:
```suggestion
```

**Pull request merged**